pom.xml

```diff
@@ -1654,6 +1655,9 @@
                 <arquillian.wildfly.jvm.args.java-version>
                     -Djdk.attach.allowAttachSelf=true
                 </arquillian.wildfly.jvm.args.java-version>
+                <pax-exam.jvm.args.java-version>
+                    --add-modules=java.xml.bind
+                </pax-exam.jvm.args.java-version>
```

Comment:
I couldn't find a way to make the JAXB dependency work in an OSGi environment, so I went this way. I'm aware it won't work for JDK 11. I created [HSEARCH-3380](https://hibernate.atlassian.net/browse/HSEARCH-3380) to address that later, but in the meantime we will at least have a proper JDK 10 build (it's about time, isn't it? :) ).

Reply:
OK, let's do it that way then. Maybe we should try to discuss it with the Karaf team in advance so that we are not stuck when JDK 11 is here (it's supposed to be released at the end of the month)?

Reply:
Sure, I'll plan that for the next sprint.